app/models/user.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const bcrypt = require('bcryptjs');
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    name: DataTypes.STRING,
+    surname: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: {
+      type: DataTypes.STRING(512),
+      set(password) {
```

Comment:
I think it is better to have this somewhere else (bcrypt encoding), specially because of further password comparison for login.

Reply:
If i create a prototype in the user model, for example User.prototype = function comparePassword(password) that returns true or false?